docs/references/backend/overview.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: Accessing the API
+description: 
+---
+
+# Accessing the API
+
+The Clerk Next.js and Node.js SDK's mirror the [Backend API](https://github.com/clerkinc/javascript/tree/main/packages/backend/src/api/endpoints). Provided is a list of the available Clerk instance methods.
+
+<Callout>
+  The backend API is accessible through the `clerkClient` object if you are using Next.js, or the `clerk` object if you are using Node.js.
+</Callout>
```

Comment:
```suggestion
The [Clerk Backend SDK](https://github.com/clerkinc/javascript/tree/main/packages/backend) exposes Clerk's backend API resources and low-level authentication utilities for JavaScript environments. While the backend SDK is mainly used as a building block for Clerk's higher-level SDKs, it can also be used on its own for advanced flows and custom integrations.

<Callout type="info">
Clerk's Next.js and Node SDKs expose an instance of the Clerk Backend SDK for use in server environments. The Backend SDK is accessible via the `clerkClient` object if you are using Next.js, and the `clerk` object if you are using Node.
</Callout>
```